tests/lib/assert.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  _assert (value, msg) {
```

Comment:
nit: it's also useful to have a basic assert function exposed publicly. having the underscore here usually denotes that the file is private.